### DISCUSSION TITLE
bluescsi mac se error

### DISCUSSION THREAD

**Discussion by pekeciel** [erichelgeson/BlueSCSI-v1#256] [General]
hi to all. today i assembly my bluescsi 1.1 internal for my mac se and green light flashes and then red light turn on. mac show a sad mac icon

i got this log file on the sd:
```
BlueSCSI https://github.com/erichelgeson/BlueSCSI
VER: 1.1-20230708-USB
DEBUG:0
SDCard Info:
 Format:exFAT
SPI speed: 50Mhz
 Max Filename Length:64
 MID:41 OID:42
 Name:SD16G Date:6/2015
 Serial:1263627723
 - hd10-512-machd.hda HDD / 41943040bytes / 40960KiB / 40MiB
ID:LUN0:
 0:----:
 1: 512:
 2:----:
 3:----:
 4:----:
 5:----:
 6:----:
Finished configuration - Starting BlueSCSI
```

can somebody gelp me?
thanks a lot

ezequiel

**Comment by erichelgeson** [comment]
Please start by taking all the steps listed on the trouble shooting guide first and let us know if that solves your issue - https://github.com/erichelgeson/BlueSCSI/wiki/Troubleshooting
